app/javascript/member-facing/track_shares.js

```diff
@@ -30,15 +31,7 @@ $(() => {
     const share = event.originalEvent.share;
     shared = true;
 
-    if (share.share_type === 'f' && window.ga) {
-      window.ga(
-        'send',
-        'event',
-        'fb:sign_share',
-        'share_progress_share',
-        window.champaign.personalization.urlParams.id
-      );
-    }
+    logEvent('social_share', share);
```

Comment:
I see you've added `share` in the case statement for event names in logEvent but don't see `social_share`. Is this meant to be `social_share`?

Reply:
You're right! Thanks for the catch. Just pushed an update to the branch.